package.json

```diff
@@ -88,6 +88,7 @@
   ],
   "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e",
   "resolutions": {
+    "@rdfjs/types": "1.1.2",
```

Comment:
I had to add this, or `clownface` (a dependency of a dependency) started to complain about the RDFJ/S types package exports. I am not sure why.